demos/demos_databases_apis/databricks_pyspark/graphistry-notebook-dashboard.ipynb

```diff
@@ -1239,23 +1217,15 @@
    },
    "outputs": [
     {
+     "output_type": "execute_result",
      "data": {
       "text/plain": [
-       "Out[18]: 'https://hub.graphistry.com/graph/graph.html?dataset=187d97493ce54498b820f727877eda4b&type=arrow&viztoken=b3106e8a-cbe9-4802-8519-97e1d0d539c3&usertag=50d9aebe-pygraphistry-0.28.5&splashAfter=1669270570&info=true&strongGravity=true'"
+       "'https://hub.graphistry.com/graph/graph.html?dataset=6d335b56affe437a98c57a349aab9e04&type=arrow&viztoken=85c14656-1740-4b16-8da0-ae1ebf138508&usertag=ac3e16de-pygraphistry-0.34.3&splashAfter=1728595501&info=true&strongGravity=true'"
```

Comment:
can you make the viz embedded, so if someone opens, they can click on it to open?

Reply:
on first skim i didn't realize there were 4 plots here!

Reply:
ah github render hides, they load fine in readthedocs: https://github.com/graphistry/pygraphistry/blob/4aa2e68ca08cc42077804707234927bf6b877ffb/demos/demos_databases_apis/databricks_pyspark/graphistry-notebook-dashboard.ipynb